SConstruct

```diff
@@ -1626,26 +1646,59 @@ if env['matlab_toolbox'] == 'y':
 # /usr/local because of dist-packages vs site-packages
 env['debian'] = any(name.endswith('dist-packages') for name in sys.path)
 
+# Check whether Cantera should be installed into a conda environment
+if conda_prefix is not None:
+    if env["layout"] != "conda":
+        # identify options selected either on command line or in cantera.conf
+        selected_options = set(line.split("=")[0].strip() for line in cantera_conf)
+        use_conda = not selected_options & \
+            {"layout", "prefix", "python_prefix", "python_cmd"}
+        use_conda &= sys.executable.startswith(conda_prefix)
+        if use_conda:
+            env["layout"] = "conda"
+            logger.info(
+                f"Using conda environment as default 'prefix': {conda_prefix}")
+elif env["layout"] == "conda":
+    logger.error("Layout option 'conda' requires a conda environment.")
+    sys.exit(1)
+
 # Directories where things will be after actually being installed. These
 # variables are the ones that are used to populate header files, scripts, etc.
-env['prefix'] = os.path.normpath(env['prefix'])
-env['ct_installroot'] = env['prefix']
-env['ct_libdir'] = pjoin(env['prefix'], env['libdirname'])
-env['ct_bindir'] = pjoin(env['prefix'], 'bin')
-env['ct_incdir'] = pjoin(env['prefix'], 'include', 'cantera')
-env['ct_incroot'] = pjoin(env['prefix'], 'include')
+if env["layout"] == "conda":
+    env["prefix"] = os.path.normpath(conda_prefix)
+
+if env["layout"] == "conda" and os.name == "nt":
+    env["ct_libdir"] = pjoin(env["prefix"], "Library", env["libdirname"])
+    env["ct_bindir"] = pjoin(env["prefix"], "Scripts")
+    env["ct_python_bindir"] = pjoin(env["prefix"], "Scripts")
+    env["ct_incdir"] = pjoin(env["prefix"], "Library", "include", "cantera")
+    env["ct_incroot"] = pjoin(env["prefix"], "Library", "include")
+else:
+    env["prefix"] = os.path.normpath(env["prefix"])
+    env["ct_libdir"] = pjoin(env["prefix"], env["libdirname"])
+    env["ct_bindir"] = pjoin(env["prefix"], "bin")
+    env["ct_python_bindir"] = pjoin(env["prefix"], "bin")
+    env["ct_incdir"] = pjoin(env["prefix"], "include", "cantera")
+    env["ct_incroot"] = pjoin(env["prefix"], "include")
+env["ct_installroot"] = env["prefix"]
```

Comment:
Can you add a newline before this line to more clearly indicate it doesn't belong in the `else` block?